business/checkers/virtualservices/single_exported_host_checker_test.go

```diff
@@ -0,0 +1,621 @@
+package virtualservices
+
+import (
```

Comment:
I see there is no usage of the `exportTo` field in this test set. Should we at least try the feature of exporting to "."?

Reply:
Okay, I see. There is no need to use the `exportTo` explicitely because the VS are appropriately added into the ExportedVirtualServices. It make sense! :+1: